cpp/src/toppra/algorithm.cpp

```diff
@@ -13,15 +23,24 @@ PathParametrizationAlgorithm::PathParametrizationAlgorithm(
 
 ReturnCode PathParametrizationAlgorithm::computePathParametrization(value_type vel_start,
                                                                     value_type vel_end) {
+  size_t be = GetCurrentTimeUsec();
   initialize();
+  size_t en = GetCurrentTimeUsec();
+  printf("time of init:                             %fms\n", (en - be)/1000.0);
+  be = en;
```

Comment:
Can you replace `printf` by `TOPPRA_LOG_DEBUG`, here and everywhere in the code ?

Reply:
Sure.